requirements_test.txt

```diff
@@ -9,4 +8,4 @@ pytest-xdist~=3.3
 six
 # Type packages for mypy
 types-pkg_resources==0.1.3
-tox>=3
+tox>=4.6.4
```

Comment:
Why are we bumping this? Is the old version broken?

Reply:
The main reason is I wanted to invalidate the cache so that I could see that the virtual environment for the Pytest stage is using the test dependencies only and not re-using the older cache from the previous run.

So other than that, I can revert this line once I'm finished noodling, or we can bump it since its a new major version - I'm ok with either outcome.

Reply:
Ok so I think the pylint job is failing only for the context of the current branch(or caches related to this branch) because the CI is now using a cache the first commit: https://github.com/pylint-dev/pylint/commit/f89e0af6c3f50a9d2f6acd9a367a3d5e979c4d38.
This commit only contains the change to the requirements_test.txt but not the "install pre-commit" command in the venv.

Apart from that I'm not sure why the primer jobs are erroring - do you have an idea about that @DanielNoord ?